.gitmodules

```diff
@@ -16,3 +16,7 @@
 	path = submodules/xdp-for-windows
 	url = https://github.com/microsoft/xdp-for-windows.git
 	branch = release/1.1
+[submodule "openssl"]
+	path = submodules/openssl
+	url = https://github.com/openssl/openssl
+	branch = master
```

Comment:
To officially support, we must point to a release branch, not mainline.

Reply:
I'll update this as soon as 3.5 releases, should be this coming tuesday